markets/perps-market/test/integration/bootstrap/bootstrapPerpsMarkets.ts

```diff
@@ -152,9 +175,26 @@ export const bootstrapPerpsMarkets: BootstrapPerpsMarketType = (data, chainState
         });
       }
 
+      let strategyId: ethers.BigNumber;
+      // create default settlement strategy
+      before('create default settlement strategy', async () => {
+        const strategy = {
+          ...DEFAULT_SETTLEMENT_STRATEGY,
+          ...(settlementStrategy ?? {}),
```

Comment:
this is where you can pass in a settlementStrategy and it'll override the defaults